test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -903,12 +940,14 @@ func TestNormalizeScorePlugin(t *testing.T) {
 		t.Errorf("Expected the pod to be scheduled. error: %v", err)
 	}
 
+	scoreWithNormalizePlugin.mutex.Lock()
```

Comment:
If pod is scheduled, I think no need of the mutex anymore, the only update to the `numScoreCalled` is scoring, and we can ignore reset here.

Reply:
I prefer to be thorough and do locking even if not needed. That's much easier to review for correctness than "that other code shouldn't get called in parallel". It's not performance sensitive.

Reply:
So if I understand you correctly, you want to have get/set methods that handle the locking?

Reply:
Yes, any concerns?

Reply:
Seems overkill and for that to be effective (i.e. no way to bypass the functions and use the fields directly) the fake plugins would have to be in a different package. But I can add it.

Reply:
As clean code, I think we should for not exposing too much internal details and mutexes are now across the test cases, hard to maintain I think.